source/package.json

```diff
@@ -2,49 +2,28 @@
   "name": "subtheme",
   "description": "Subtheme file for us in KIT-related projects",
   "author": "",
-  "version": "8.1.2",
+  "version": "1.0.0",
```

Comment:
I feel like we should make this 8.2.0, to follow convention that it's made for Drupal 8. Similarly, I was thinking I'd make our new tools be 8.2 as well, to make sure that we have a deprecated tool set for all of the current sites.